### DISCUSSION TITLE
Support to have a branch other than default branch for Pull requests

### DISCUSSION THREAD

**Discussion by behitesh** [community/community#14464] [Pull Requests]
> The default branch is considered the “base” branch in your repository, against which all pull requests and code commits are automatically made unless you specify a different branch.

Currently, we have an option to set any branch as the default branch.

## Would love to see this...

Ability to configure say... `main` as the default branch and have `develop` as the branch against all pull requests are made.

with this... we can have all folks start out with a more stable branch - `main` in this case... But when raising a PR the base branch defaults to another branch - `develop` where all the active changes to be tested are merged.

**Comment by tounsoo** [comment]
This would be awesome
